src/app/state/manager.rs

```diff
@@ -695,14 +695,27 @@ impl State for ManagerCreateSendTransactionState {
         match &mut self.step {
             ManagerSendStep::WelcomeUser(v) => v.view(),
             ManagerSendStep::SelectOutputs(v) => {
-                let valid = !self.outputs.is_empty() && !self.outputs.iter().any(|o| !o.valid());
+                let mut valid =
+                    !self.outputs.is_empty() && !self.outputs.iter().any(|o| !o.valid());
+                // TODO: remove this check once revaultd accept vector of recipients instead of map
+                let mut no_duplicate = true;
+                for (i, output) in self.outputs.iter().enumerate() {
+                    if self.outputs[i + 1..]
```

Comment:
Hmm how is it not an overflow?

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=93ef35e3d58294287a5943c033e7ad4b
the [..] do the trick and return empty slice

Reply:
So weird. Cool then!